### PR TITLE
Remove scrollbars on Identify2 popup

### PR DESCRIPTION
**Pull request by zanderz**
@cecileboucheron said we could just take off the scrollbars, like the original Identify.

### CODE REVIEW

**Comment by chrisnojima** [comment]
Let's talk to design

**Comment by zanderz** [comment]
@keybase/react-hackers Cecile said we could remove the scrollbars, this is ready

File: shared/common-adapters/scroll-view.desktop.js [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
 }
 
-const styles = styleSheetCreate({
-  overflowAuto: platformStyles({
-    isElectron: {
-      overflow: 'auto',
-    },
-  }),
-})
+// not using a stylesheet because StylesCrossPlatform
```
Comment (chrisnojima) [inline shared/common-adapters/scroll-view.desktop.js]: We should do the opposite. If it's missing add it to the desktop styles definition

Comment (zanderz) [reply shared/common-adapters/scroll-view.desktop.js]: ok got it

**Comment by zanderz** [comment]
@chrisnojima how about now?

**Comment by chrisnojima** [comment]
fyi overflowx/y isnt' valid on RN, neither is auto

**Pull request merged**